cmd/loom/chainconfig/cmd.go

```diff
@@ -502,6 +503,10 @@ func SetSettingCmd() *cobra.Command {
 				return invalidValueTypeError
 			}
 
+			if args[0] != "AppStore.NumEvmKeysToPrune" {
+				return invalidSettingName
+			}
```

Comment:
That's not what I had in mind, I don't want to modify this command every time we change the config schema to add/remove a field. This should do the reflection based lookup & value assignment that happens when the setting change is actually applied by a node.

Reply:
Yeah actually, we can just use the setting function to validate the arguments, we don't really need a type flag. Fixed.